requirements-dev.txt

```diff
@@ -5,3 +5,4 @@ ipdb==0.12
 pip-tools==3.7.0
 pre-commit==1.17.0
 tox==3.11.1
+mock==4.0.2
```

Comment:
Note this will be resolved via https://github.com/druid-io/pydruid/pull/190.

Reply:
👌 I'll remove it.